eng/scripts/Language-Settings.ps1

```diff
@@ -15,7 +15,15 @@ function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgNa
     $jsStylePkgName = $projectJson.name.Replace("@", "").Replace("/", "-")
     if ($pkgName -eq "$jsStylePkgName")
     {
-      return [PackageProps]::new($projectJson.name, $projectJson.version, $pkgPath, $serviceDirectory)
+      $pkgProp = [PackageProps]::new($projectJson.name, $projectJson.version, $pkgPath, $serviceDirectory)
+      $pkgProp.SdkType = $projectJson.psobject.properties['sdk-type'].value
+      if ($projectJson.name.StartsWith("@azure/arm"))
+      {
+        $pkgProp.SdkType = "mgmt"
+      }
+      $pkgProp.IsNewSdk = $pkgProp.SdkType -eq "client"
```

Comment:
@praveenkuttappan As discussed before, the `@azure/event-processor-host` package does have the sdk type set to client, but is not a Track 2 package. So, you may want to filter that out here.

Reply:
@ramya-rao-a we should avoid needing to have these special filters in various places (it wouldn't just be here). The source of truth is the sdk-type property in the project and if we want this package to not be treated like a client package we need to change that property in that project.

Reply:
@weshaggard, `@azure/event-processor-host` is indeed a client package and uses rush and uses our release pipeline, it is just not Track 2. So, if you hope to use this script to differentiate between Track 1 and Track 2 packages, then you will get a false positive

Reply:
In most other contexts across all our repo's, including JS, the term "client" generally refers to new track 2 libraries. For track 1 libraries in most other languages we usually referred to them as "data" just to keep the identifier unique.

Reply:
@ramya-rao-a: On a similar note, I have found that SDK type is client for a utility proejct. @azure/test-utils-recorder 
Wondering if this is intentional and if there is any reasoning behind it? Shouldn't we set these as util type for any of these internal tools?

Reply:
cc @HarshaNalluru

Reply:
`test-utils-recorder` was the first "utility" project in the js repo, and we didn't make the distinction at the time of its creation.

We can mark it as "utility".. 
But I do remember seeing `rush-runner.js` checking if the sdk-type is a client or not in some of its methods, we just need to make sure the package gets built and the existing pipelines(live/unit test) don't break with this change.

Reply:
And btw, perf framework is also marked as the "client", we should handle that too.

Reply:
@praveenkuttappan As a follow up on this, please log an issue to review all existing cases of sdk-type = client